index.html

```diff
@@ -41,24 +41,24 @@
         </p>
 
         <script>
-var FIRST_REORG_DATE = new Date('2015-03-27');
-var LAST_REORG_DATE = new Date('2015-06-05');
-var NOW = new Date();
+var FIRST_REORG_TIME = +new Date('2015-03-27');
+var LAST_REORG_TIME = +new Date('2015-06-05');
+
+var NOW = +new Date();
 var ONE_DAY = 24 * 60 * 60 * 1000;
 
+var jquery = document.getElementById.bind(document);
```

Comment:
Maybe we should add a factory so that people could decide to use either jquery or zepto?